src/main/java/org/apache/commons/imaging/formats/tiff/constants/MicrosoftHdPhotoTagConstants.java

```diff
@@ -152,17 +152,17 @@
     public static final BinaryConstant PIXEL_FORMAT_VALUE_32_BIT_GRAY_FIXED_POINT =
             createMicrosoftHdPhotoGuidEndingWith((byte) 0x3f);
 
-    public static final TagInfoLong EXIF_TAG_TRANSFOMATION = new TagInfoLong(
-            "Transfomation", 0xbc02,
+    public static final TagInfoLong EXIF_TAG_TRANSFORMATION = new TagInfoLong(
+            "Transformation", 0xbc02,
             TiffDirectoryType.EXIF_DIRECTORY_UNKNOWN);
-    public static final int TRANSFOMATION_VALUE_HORIZONTAL_NORMAL = 0;
-    public static final int TRANSFOMATION_VALUE_MIRROR_VERTICAL = 1;
-    public static final int TRANSFOMATION_VALUE_MIRROR_HORIZONTAL = 2;
-    public static final int TRANSFOMATION_VALUE_ROTATE_180 = 3;
-    public static final int TRANSFOMATION_VALUE_ROTATE_90_CW = 4;
-    public static final int TRANSFOMATION_VALUE_MIRROR_HORIZONTAL_AND_ROTATE_90_CW = 5;
-    public static final int TRANSFOMATION_VALUE_MIRROR_HORIZONTAL_AND_ROTATE_270_CW = 6;
-    public static final int TRANSFOMATION_VALUE_ROTATE_270_CW = 7;
+    public static final int TRANSFORMATION_VALUE_HORIZONTAL_NORMAL = 0;
+    public static final int TRANSFORMATION_VALUE_MIRROR_VERTICAL = 1;
+    public static final int TRANSFORMATION_VALUE_MIRROR_HORIZONTAL = 2;
+    public static final int TRANSFORMATION_VALUE_ROTATE_180 = 3;
+    public static final int TRANSFORMATION_VALUE_ROTATE_90_CW = 4;
+    public static final int TRANSFORMATION_VALUE_MIRROR_HORIZONTAL_AND_ROTATE_90_CW = 5;
+    public static final int TRANSFORMATION_VALUE_MIRROR_HORIZONTAL_AND_ROTATE_270_CW = 6;
+    public static final int TRANSFORMATION_VALUE_ROTATE_270_CW = 7;
```

Comment:
Ouch! Good catch! Thanks

Reply:
;) just was lucky. I was checking the variable in general